material-components/material-components-web
# chore: Create script to cherry-pick commits for patch releases

To test:

* Checkout this branch
* Run `node scripts/cherry-pick-patch-commits`

It should check out the latest tag, cherry-pick commits, and run the build and unit tests locally, then provide output summarizing what was cherry-picked, what was skipped, and what was conflicted, and whether the build and tests succeeded. You will then be on a detached HEAD with cherry-picked commits on top of the tag you selected. None of these cherry-picked commits should be features or breaking changes.

For more of a stress-test, you can also pass it a previous release tag, e.g. `node scripts/cherry-pick-patch-commits v0.35.2`. (This will fail the build, which is expected, because there is a fix that depends on a feature; the fix was cherry-picked but the feature wasn't.)
